src/citrine/seeding/find_or_create.py

```diff
@@ -15,9 +15,11 @@ def find_collection(collection, name):
             # call list() to collapse the iterator, otherwise the NotFound
             # won't show up until collection_list is used
             collection_list = list(collection.search(search_params={
-                "name": {
-                    "value": name,
-                    "search_method": "EXACT"
+                "search_params": {
```

Comment:
I think nesting a "search_params" key inside an argument called "search_params" is very confusing UX.  Could this be fixed by adjusting the logic in the search method [here](https://github.com/CitrineInformatics/citrine-python/blob/master/src/citrine/resources/project.py#L469) ?

Reply:
In retrospect, I probably should have changed the ux for that - i was keeping it the same as what the product api request expects. I'll change it